psi4/driver/p4util/python_helpers.py

```diff
@@ -991,13 +991,22 @@ def _core_wavefunction_set_frequencies(cls, val):
 
 def _core_wavefunction_X(cls):
     warnings.warn(
-        "Using `psi4.core.Wavefunction.X` instead of `psi4.core.Wavefunction.lagrangian` is deprecated, and in 1.4 it will stop working\n",
+        "Using `psi4.core.Wavefunction.X` instead of `psi4.core.Wavefunction.lagrangian` is deprecated, and in 1.5 it will stop working\n",
+        category=FutureWarning,
+        stacklevel=2)
+    return cls.lagrangian()
+
+
+def _core_wavefunction_Lagrangian(cls):
```

Comment:
thanks, but I don't think py-side Lagrangian deprecation is needed since the fn didn't exist two weeks ago: https://github.com/psi4/psi4/blob/v1.4a2/psi4/src/export_wavefunction.cc . am I missing something?

Reply:
No, my bad. I removed it.